File: dataset/human/c/file_9463.c
#include <stdio.h>

int main(void) {
	int N, D, X;
	int A[100];
	int count = 0;
	scanf("%d%d%d", &N, &D, &X);
	for (int i = 0; i < N; i++) {
		scanf("%d", &A[i]);
	}

	for (int i = 0; i < N; i++) {

		for (int j = 0; j < D; j++) {
			if (D>= j * A[i] + 1) {

				count++;

			}
		}
	}



	printf("%d\n", count + X);



	return 0;

}

File: dataset/human/java/file_9463.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int count = 0;
		for (int i = 0; i < N; i++) {
			int a = sc.nextInt();
			int b = sc.nextInt();
			if (a == b) {
				count++;
				if (count >= 3) {
					System.out.println("Yes");
					return;
				}
			} else {
				count = 0;
			}
		}

		System.out.println("No");
	}

}
